<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectReferencesWidget</class>
 <widget class="QWidget" name="ObjectReferencesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="references_grid">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="object_lbl">
       <property name="text">
        <string>Object:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sel_obj_icon_lbl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="ref_name_lbl">
     <property name="text">
      <string>Ref. name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QLineEdit" name="ref_name_edt">
       <property name="toolTip">
        <string>&lt;p&gt;The name of the reference to an object. All occurences of the reference enclosed by &lt;strong&gt;{}&lt;/strong&gt; are replaced by the referenced object's name or signature in the generic SQL object's code.&lt;/p&gt;</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ref_alias_lbl">
       <property name="text">
        <string>Ref. alias:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ref_alias_edt">
       <property name="toolTip">
        <string>&lt;p&gt;The name of the reference to an object. All occurences of the reference enclosed by &lt;strong&gt;{}&lt;/strong&gt; are replaced by the referenced object's name or signature in the generic SQL object's code.&lt;/p&gt;</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ref_name_lbl_3">
     <property name="text">
      <string>Options:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QCheckBox" name="format_name_chk">
       <property name="toolTip">
        <string>&lt;p&gt;The referenced object's name or signature will be automatically quoted when special characters are found. Additionally, for schema qualified objects, the name of the parent schema is prepended to the referenced object's name or signature. This will avoid common SQL syntax errors or loss of semantics.&lt;/p&gt;</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>Format name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="use_signature_chk">
       <property name="toolTip">
        <string>&lt;p&gt;Use the referenced object's signature instead of its name. For some objects like functions, casts, operators and some others the signature will include parameters types and some other information.&lt;/p&gt;</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>Use signature</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
